Dockerfile

```diff
@@ -21,7 +21,7 @@ ENV LANG=en_US.UTF-8 \
 RUN apt-get update && \
     apt-get install -y gcc git graphviz make rsync ssh vim-tiny wget && \
     ln -s /usr/bin/vi /usr/bin/vim && \
-    wget --content-disposition "http://packages.chef.io/files/${CHANNEL}/chef-workstation/${VERSION}/ubuntu/18.04/chef-workstation_${VERSION}-1_amd64.deb" -O /tmp/chef-workstation.deb && \
+    wget --content-disposition "http://packages.chef.io/files/${CHANNEL}/chef-workstation/${VERSION}/ubuntu/22.04/chef-workstation_${VERSION}-1_amd64.deb" -O /tmp/chef-workstation.deb && \
```

Comment:
the URL where package is maintained continues to be 18.04. We are yet to shift to building package on newer Ubuntu version. this line change would need to be discarded.

Reply:
Change in ba09e683

Reply:
@vkarve-chef changed in 2947b0bd